src/components/PathwayExercise/ExerciseContent/Assessment/index.js

```diff
@@ -0,0 +1,211 @@
+import React, { useState } from "react";
+import { Container, Box, Typography, Paper, Button } from "@mui/material";
+import useStyles from "./styles";
+import get from "lodash/get";
+import DOMPurify from "dompurify";
+
+const headingVarients = {};
+
+[Typography, "h2", "h3", "h4", "h5", "h6"].forEach(
+  (Name, index) =>
+    (headingVarients[index + 1] = (data) => (
+      <Name
+        className="heading"
+        dangerouslySetInnerHTML={{ __html: data }}
+        {...(index === 0 ? { component: "h1", variant: "h6" } : {})}
+      />
+    ))
+);
+
+const AssessmentContent = ({
+  content,
+  answer,
+  setAnswer,
+  setSolution,
+  submit,
+  correct,
+  index,
+  setSubmitDisable,
+}) => {
+  const classes = useStyles();
+  if (content.component === "header") {
+    return headingVarients[content.variant](
+      DOMPurify.sanitize(get(content, "value"))
+    );
+  }
+  if (content.component === "text") {
+    const text = DOMPurify.sanitize(get(content, "value"));
+    if (index === 0) {
+      setSubmitDisable(true);
+      return (
+        <Box sx={{ mt: "32px" }}>
+          <Typography variant="h6" align="center">
+            Output
+          </Typography>
+
+          <Box
+            sx={{
+              mt: "32px",
+              bgcolor: correct ? "#E9F5E9" : "#FFE5E3",
```

Comment:
I'd recommend getting these colors from the theme, so you don't have to replace all occurrences if the theme colors were to ever change.  If you always want them to match `success` and `error` colors, you could use the following:
```suggestion
              bgcolor: correct ? "success.light" : "error.light",
```